go.mod

```diff
@@ -1,20 +1,23 @@
 module github.com/fission/fission
 
+go 1.15
```

Comment:
Hi @tosone We have kept the Fission GO version for now at 1.12.17. Also, I don't understand what this PR is trying to do, can you please explain?

Reply:
Emm, I will replace `go 1.15` with `go 1.12`.

File `github.com/ory/dockertest/docker/pkg/term/tc.go` line 13 import `unix.SYS_IOCTL`, library `golang.org/x/sys/unix` is too old, SYS_IOCTL, `github.com/ory/dockertest/docker/pkg/term/tc.go` build on non-windows, on my mac it will be an error.

Reply:
I search some history, `unix.SYS_IOCTL` defines in `golang.org/x/sys/unix/zsysnum_darwin_amd64.go` line 54.
This file was removed in [commit](https://github.com/golang/sys/commit/6fcdbc0bbc04dcd9e7dc145879ceaf9bf1c6ff03#diff-4c0bfb5f8598d17f4158c9aaa2833ea8672bd71131cdb2a5a8e64a93e67b83d3), after that added by [commit](https://github.com/golang/sys/commit/c5abc1b1d397186b106fe10bd156adac799e8598#diff-4c0bfb5f8598d17f4158c9aaa2833ea8672bd71131cdb2a5a8e64a93e67b83d3) here.